src/Uno.UI/Uno.UI.Skia.csproj

```diff
@@ -11,6 +15,7 @@
 		<NoWarn>$(NoWarn);NU1701</NoWarn>
 		<DefineConstants>$(DefineConstants);IS_UNO;IS_UNO_UI_PROJECT</DefineConstants>
 		<AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
There is a `<WarningsAsErrors>true</WarningsAsErrors>` line below. It should be removed.